Writerside/topics/conclusion-and-outlook.md

```diff
@@ -1,3 +1,28 @@
 # Conclusion and Outlook
 
-Concluding it can be stated, that all must-have requirements are fully satisfied.
\ No newline at end of file
+In conclusion, the project successfully fulfilled all the must-have, most of the should-have, and some of the
+nice-to-have requirements, delivering a production-ready transit routing solution.
+
+One of the key strengths of this project is that the final product is available as open-source under the MIT license,
```

Comment:
Is this our primary strength? It seems like it would be very easy to have a successful project if just making it open source is the main achievement regardless of how bad the product is :).

I would start with more code related strengths, such as how we were able to extend the raptor algorithm to use more schedule specific information, multi-day. Requirements that were only identified within the project --> leading into flexibility / iterative approach in project planning.

Reply:
https://github.com/naviqore/documentation/pull/25/commits/a09551a98ffaf6998e867a92b085491d3b981d40